examples/README.md

```diff
@@ -3,33 +3,30 @@
 This is a collection of different ways to run Mighty. It's not a full documentation of all features or possibilities, but rather documenting different entry points into the code base.
 Here's an overview of example content:
 
-- [Mighty Examples](#mighty-examples)
-  - [Running Mighty from the Command Line](#running-mighty-from-the-command-line)
+- [Running Mighty from the Command Line](#running-mighty-from-the-command-line)
     - [The Runner File](#the-runner-file)
     - [Setting the Environment](#setting-the-environment)
     - [Configuring Agents](#configuring-agents)
     - [Meta Components](#meta-components)
     - [Running Multiple Seeds](#running-multiple-seeds)
-  - [Building Mighty Hydra Configs](#building-mighty-hydra-configs)
-    - [Configuration Basics](#configuration-basics)
-    - [Considerations for configuration stacking](#considerations-for-configuration-stacking)
-  - [Hyperparameter Optimization Options](#hyperparameter-optimization-options)
-    - [Hydra Sweepers](#hydra-sweepers)
-    - [Hypersweeper](#hypersweeper)
-    - [ES Runner](#es-runner)
-  - [Adding Custom Mighty Components](#adding-custom-mighty-components)
+- [Adding Custom Mighty Components](#adding-custom-mighty-components)
     - [The Metrics Dictionary](#the-metrics-dictionary)
     - [Priority Flexibility: Meta-Components](#priority-flexibility-meta-components)
     - [Building a Custom Component](#building-a-custom-component)
-  - [Logging \& Plotting](#logging--plotting)
-
+- [Hyperparameter Optimization Options](#hyperparameter-optimization-options)
+    - [The Mighty ES Runner](#the-mighty-es-runner)
+    - [Hydra Sweepers](#hydra-sweepers)
+    - [Hypersweeper](#hypersweeper)
+- [Logging \& Plotting](#logging--plotting)
 
 ## Running Mighty from the Command Line
 
 Let's start with the basics, running a Mighty run from the command line! Mighty uses [Hydra](https://hydra.cc/) as a command line interface, so having basic familiarity with it will make your life easier.
+These examples show you how to configure Mighty from the command line, if you're interested in how to build configuration files, check out [our short guide](./Building_mighty_configs.md).
 
-### The Runner File
-
+#### The Runner File
+<details open>
```

Comment:
Not sure if we need the details being open from the start. I'd expect most people to use run_mighty.py anyways.